src/adv.py

```diff
@@ -33,12 +41,25 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
-#
+room['treasure'].item = item['gold']
+room['outside'].item = item['rock']
+
+
+def move_to(direct, location):
```

Comment:
Try to think about this in a more object oriented way. How else could a player move, without the use of a function?